dev/com.ibm.ws.container.service/src/com/ibm/ws/container/service/annocache/package-info.java

```diff
@@ -0,0 +1,13 @@
+/*******************************************************************************
+ * Copyright (c) 2011, 2015 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+/** @version 3.0 */
+@org.osgi.annotation.versioning.Version("3.0")
```

Comment:
why is a new package starting at 3.0?

Reply:
Yeah; that doesn't make sense.  I've updated that version and the version of internal/package-info.java to 1.0.